pkgs/by-name/je/jellyfin-tui/package.nix

```
@@ -0,0 +1,35 @@
+{
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+  pkg-config,
+  openssl,
+  mpv,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "jellyfin-tui";
+  version = "v1.0.5";
+
+  src = fetchFromGitHub {
+    owner = "dhonus";
+    repo = pname;
+    rev = version;
```

Use the modern version of specifying the version when fetching tags.
```suggestion
    tag = version;
```